GetStream/stream-chat-android
# [3575] Eliminate deperecated is admin checks

### 🎯 Goal

Eliminate `Member.isOwnerOrAdmin` and `List<Member>?.isCurrentUserOwnerOrAdmin` usages

### 🛠 Implementation details

Replaced `Member.isOwnerOrAdmin` and `List<Member>?.isCurrentUserOwnerOrAdmin` with `ownCapabilities` checks.

### 🧪 Testing

1. Run on develop and login as Jc, check rights to delete Uniform Annual and the rights to remove other channels members
2. Login as member and check if the rights are still there
3. Run on this branch and do step 1 and 2 again

### ☑️Contributor Checklist

#### General

#### Code & documentation

### ☑️Reviewer Checklist

### 🎉 GIF

![giphy](https://user-images.githubusercontent.com/38032787/172874563-60b87599-bec5-463b-bcbd-ad89ea945d52.gif)
